packages/vats/src/core/lib-boot.js

```diff
@@ -74,19 +71,14 @@ const buildRootObject = (vatPowers, vatParameters) => {
   const rawBootstrap = async (vats, devices) => {
     // Complete SwingSet wiring.
     const { D } = vatPowers;
-    if (devices.mailbox) {
```

Comment:
hm. why did I take this out? I wonder if it was on purpose.

Can someone remind me why it was added?

Reply:
I don't remember exactly why this check was added.  It seems that sim, solo, and real chain all have a mailbox device.  Maybe this was just extra defensiveness.

Reply:
I can look it up in git... maybe it's coming back to me, meanwhile: I think it was for the vaults + smart wallet bootstrap test.